modules/model/item/components/overcast.js

```diff
@@ -80,11 +87,11 @@ export class OvercastItemModel extends BaseItemModel {
   // Don't really like this here as it uses assumed subclass data, but it'll do for now
   computeOvercastingData() {
     let usage = {
-      damage: undefined,
-      range: undefined,
-      duration: undefined,
-      target: undefined,
-      other: undefined,
+      damage: null,
+      range: null,
+      duration: null,
+      target: null,
+      other: null,
```

Comment:
Because it's in schema now, it can't be `undefined`, so I changed to `null`. Doesn't impact anything.

Anyway, as above I'm not 100% sure yet this is needed and might revert this if not needed.